TinyOS/Assembler1.s

```diff
@@ -167,17 +174,19 @@ UART_Put:
 
 .global ADC_Get
 ADC_Get:
-		ldi		r16,0xC7			//Load 199 into r16
-		sts		ADCSRA,r16			//Move 199 from r16 to ADC Control and Status Register A
+		ldi		r16,0xE7			//Load 231 into r16
```

Comment:
And an explanation for this value/address?

Reply:
Loading the proper flags into the ADCSRA